test/cli/test_all.rb

```diff
@@ -0,0 +1,2 @@
+# NOTE: run the test suite with: $ bundle exec ruby -I lib:test test/cli/test_all.rb -v
+require_relative "test_kbsecret.rb"
```

Comment:
Minor nit: Drop the `.rb` suffix here for consistency with the rest of the project.